### PR TITLE
pipename begins with fix and new havoc c2 pipenames

### PR DESCRIPTION
**Pull request by pH-T**

### CODE REVIEW

**Comment by phantinuss** [comment]
We need a discussion about `\mojo.`. It will match on the legitimate chrome named pipe and as id 17+18 are configured together and id 18 is on pipe connection we might get lots of log data enabling it. Not adding it we have less visibility for malformed `\mojo.` pipe names.

My opinion is to not add `\mojo.`.
Are there other opinions? @Neo23x0 @humpalum

**Comment by Neo23x0** [comment]
Yeah, we should leave it out of the config

File: sysmonconfig-export-block.xml [hunk 1]
```diff
@@ -918,13 +918,16 @@
...
+				<PipeName condition="begin with">\mojo_</PipeName>
+				<PipeName condition="begin with">\interprocess_</PipeName>
+				<PipeName condition="begin with">\samr_</PipeName>
+				<PipeName condition="begin with">\netlogon_</PipeName>
+				<PipeName condition="begin with">\srvsvc_</PipeName>
+				<PipeName condition="begin with">\lsarpc_</PipeName>
+				<PipeName condition="begin with">\wkssvc_</PipeName>
+				<!-- Havoc C2 default -->
+				<PipeName condition="begin with">\demon_pipe</PipeName>
+				<PipeName condition="begin with">\mojo.</PipeName>
```
Comment (phantinuss) [inline sysmonconfig-export-block.xml]:
```suggestion
```

File: sysmonconfig-export.xml [hunk 2]
```diff
@@ -960,13 +960,16 @@
...
+				<PipeName condition="begin with">\mojo_</PipeName>
+				<PipeName condition="begin with">\interprocess_</PipeName>
+				<PipeName condition="begin with">\samr_</PipeName>
+				<PipeName condition="begin with">\netlogon_</PipeName>
+				<PipeName condition="begin with">\srvsvc_</PipeName>
+				<PipeName condition="begin with">\lsarpc_</PipeName>
+				<PipeName condition="begin with">\wkssvc_</PipeName>
+				<!-- Havoc C2 default -->
+				<PipeName condition="begin with">\demon_pipe</PipeName>
+				<PipeName condition="begin with">\mojo.</PipeName>
```
Comment (phantinuss) [inline sysmonconfig-export.xml]:
```suggestion
```

**Comment by pH-T** [comment]
@phantinuss fixed

**Pull request merged**